src/app/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -249,23 +249,23 @@ const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModalProps>
                 {fields?.hostname ? (
                   <ValidatedTextInput
                     field={fields.hostname}
-                    label="Hostname or IP address"
+                    label="RHV Manager Hostname or IP address"
```

Comment:
Following Patternfly4 and RHV conventions, this should be "RHV Manager host name or IP address"